Makefile

```
@@ -143,7 +145,7 @@ run: manifests generate fmt vet ## Run a controller from your host.
 .PHONY: docker-build
 docker-build: test ## Build docker image with the manager.
 	docker build \
-		-t ${IMG} \
+		-t ${IMG} $(IMAGE_AUTH) \
```

The meaning of IMAGE_AUTH is is a bit unclear. It exposes a direct dependency to the docker command line for no good reason. I'd prefer IMAGE_AUTH to be called something like AUTH_FILE and do
```suggestion
		-t ${IMG} \
		--authfile $(AUTH_FILE) \
```

---

Changed to AUTH_FILE 
Explanation is in DEVELOPMENT.md